server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -230,7 +231,8 @@ private Metadata(
         String[] visibleOpenIndices,
         String[] allClosedIndices,
         String[] visibleClosedIndices,
-        SortedMap<String, IndexAbstraction> indicesLookup
+        SortedMap<String, IndexAbstraction> indicesLookup,
+        Map<String, MappingMetadata> cache
```

Comment:
NIT: Maybe call this something mapping specific, like `mappingsByHash` or so? :) `cache` seems a little generic :)

Reply:
I've renamed this: 3bd8a655ddc506f88297f79d9339138cd198f73b